### ISSUE TITLE
Add build step

### ISSUE THREAD

**Issue by mike182uk** [gitevents/gitevents-tito#13]
We should consider having a build step that does things like check code-style, runs tests etc. and provides feedback on new pull requests.

**Comment by PatrickHeneise** [comment]
code-style can be checked by hound, tests by codeship etc. Most of the services run on the core already, we can just include the plugin repos?

**Comment by iancrowther** [comment]
@mike182uk - please check out http://jscs.info/

**Comment by iancrowther** [comment]
eslint also

**Comment by mike182uk** [comment]
Yep i am aware of these tools, the issue was more aimed at using CI to run these things automatically on a new PR. Hound and Codeship would work, and if they are already in use on core then it shouldn't be too difficult to integrate them into this.

**Comment by iancrowther** [comment]
Can switch on, do you have permissions

**Comment by PatrickHeneise** [comment]
hound currently runs an old version of jshint. We could turn hint off and use jscs only?

**Comment by iancrowther** [comment]
happy to go that route

**Comment by PatrickHeneise** [comment]
Just tried it in a recent project, unfortunately jscs really does only code style and doesn't tell you about important JS issues like jshint does.
